MultiTable/lens.json

```diff
@@ -36,6 +36,7 @@
     "1.0.18": "Show aspect help links and related links in sample modal",
     "1.0.19": "Aspect name case insensitivity - missed one",
     "1.0.20": "Add google analytics tracking for search, onclick modals and links, and toggle checkbox",
-    "1.0.21": "Prevent reverse tabnabbing"
+    "1.0.21": "Prevent reverse tabnabbing",
+    "1.0.22": "Add RealtimeChangeHandler arg validation and tests"
```

Comment:
If everyone's going to be making lots of concurrent changes, maybe we should hold off on updating this until all the changes are in.

Reply:
I'm OK with that, as long as the "last one in" adds meaningful release notes to the lens.json.